### ISSUE TITLE
Works on Windows?

### ISSUE THREAD

**Issue by johnbent** [hpc/ior#41]
Has anyone run IOR/mdtest on Windows?  Any tricky bits?

**Comment by ezickler** [comment]
No. But if some one figures it out, would be great to have a CI set up for windows with https://www.appveyor.com/. So we can detect regression on windows, as i suspect this will be used much less often then on linux.
